docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f8b7f2dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install cleanlab termcolor "
```

Comment:
change the installation of cleanlab to be pip installed from your branch instead for now. 
This way you won't have to use hacks like importing from `rank` module rather than `cleanlab.token_classification.rank`

https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch

Reply:
It's not working for me, need your help later :(